raiden/transfer/events.py

```diff
@@ -45,6 +45,14 @@ class SendWithdrawRequest(SendMessageEvent):
     expiration: BlockExpiration
     nonce: Nonce
 
+    def __repr__(self) -> str:
+        return (
+            f"SendWithdrawRequest< "
```

Comment:
```suggestion
            f"{self.__class__.__name__}< "
```
would allow for easier refactoring in the future

Reply:
Very good point.